compute_endpoint/globus_compute_endpoint/endpoint/config/config.py

```diff
@@ -172,8 +172,8 @@ def __init__(
         self.environment = environment
         self.funcx_service_address = funcx_service_address
 
-        self.multi_tenant = multi_tenant is True
-        self.force_mt_allow_same_user = force_mt_allow_same_user is True
+        self.multi_user = multi_user is True
+        self.force_mu_allow_same_user = force_mu_allow_same_user is True
```

Comment:
This enforces that only boolean `True` -- not truthy values -- is accepted. However, the test suite only parametrizes `(None, True, False)`, and uses conditions like `if mu:` and `if mu is not None:` to guard against certain code paths in the test suite.

You might consider adding an additional parameter that is truthy but not boolean `True` -- like `1` or `"bogus"` -- and ensuring that `.multi_user` and `.force_mu_allow_same_user` are _not_ enabled in those circumstances. Doing so will differentiate between this current `multi_user is True` code and future changes to these two lines.

Reply:
This is out of scope for this PR, but the point is well taken. I'm sure the team would appreciate another PR to address this.